projects/plugins/jetpack/modules/gravatar-hovercards.php

```diff
@@ -226,15 +250,14 @@ function grofiles_get_avatar( $avatar, $author ) {
  * @todo is_singular() only?
  */
 function grofiles_attach_cards() {
-	global $blog_id;
```

Comment:
I'm not sure if I should have used the `// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable` here instead of removing the variable?

Reply:
Since the variable isn't used in the function, it's okay to remove it.